third_party/move/move-binary-format/src/check_complexity.rs

```diff
@@ -244,6 +244,7 @@ impl<'a> BinaryComplexityMeter<'a> {
                 },
                 StructFieldInformation::DeclaredVariants(variants) => {
                     for variant in variants {
+                        self.meter_identifier(variant.name)?;
```

Comment:
Wouldn't this introduce replay issues?

Reply:
As discussed on slack, no, because its not gas related.